### PR TITLE
Add nano tensorflow examples

### PR DESCRIPTION
**Pull request by MeouSker77**
## Description

* Add nano tensorflow multi-instance training example
* Add nano tensorflow sparse embedding example

### 1. Why the change?

We should add more examples to demonstrate the usage of nano.

### 2. User API changes

N/A

### 3. Summary of the change 

Add some basic tensorflow examples

### 4. How to test?
- [ ] N/A
- [x] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

N/A

### CODE REVIEW

File: python/nano/tutorial/training/tensorflow/tensorflow_train_multi_instance.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    # BigDL-Nano will launch the specific number of processes to perform data-parallel training.
+    #
+    model = create_model(num_classes=num_classes, img_size=img_size)
+    model.fit(ds_train,
+              epochs=num_epochs,
+              steps_per_epoch=steps_per_epoch,
+              validation_data=ds_test,
+              validation_steps=validation_steps,
+              num_processes=4,
+              backend='multiprocessing')
```
Comment (jason-dai) [inline python/nano/tutorial/training/tensorflow/tensorflow_train_multi_instance.py:105]: do we have to specify the backend here?

Comment (MeouSker77) [reply python/nano/tutorial/training/tensorflow/tensorflow_train_multi_instance.py]: it's not necessary

File: python/nano/tutorial/training/tensorflow/tensorflow_train_multi_instance.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    # Multi-Instance Training
+    # 
+    # It is often beneficial to use multiple instances for training
+    # if a server contains multiple sockets or many cores, 
+    # so that the workload can make full use of all CPU cores.
+    # BigDL-Nano makes it very easy to conduct multi-instance training correctly.
+    # 
+    # Use `Model` or `Sequential` in `bigdl.nano.tf.keras` to create model,
+    # then just set the `num_processes` parameter in the `fit` method.
+    # BigDL-Nano will launch the specific number of processes to perform data-parallel training.
```
Comment (jason-dai) [inline python/nano/tutorial/training/tensorflow/tensorflow_train_multi_instance.py:82]: also need to explain learning rate scaling & warmup?

Comment (MeouSker77) [reply python/nano/tutorial/training/tensorflow/tensorflow_train_multi_instance.py]: done

**Review by TheaperDeng — APPROVED** [review]
LGTM
